src/cgame/rocket/rocket_documents.cpp

```diff
@@ -58,9 +58,9 @@ void Rocket_LoadDocument( const char *path )
 }
 
 // Scale the UI proportional to the screen size
-void Rocket_SetDocumentScale( Rml::Core::ElementDocument& document )
+void Rocket_SetDocumentScale( Rml::ElementDocument& document )
 {
-	// This makes 1dp one pixel on a 1920�1080 screen
+	// This makes 1dp one pixel on a 1920�1080 screen
```

Comment:
For the second-to-last commit, I think you changed the comment to something not-unicode :S

![Screenshot_20220804-101103-1-1](https://user-images.githubusercontent.com/59283660/182798250-490cc15f-4531-4aa1-aebc-5f61fbd4bbbe.jpg)

The comment is wrong anyway, it should say 1366×768

Reply:
Fixed it.